BioArchLinux/minimap2/PKGBUILD

```diff
@@ -3,23 +3,44 @@
 
 pkgname=minimap2
 pkgver=2.26
-pkgrel=1
+pkgrel=2
 pkgdesc='A versatile pairwise aligner for genomic and spliced nucleotide sequences. https://doi.org/10.1093/bioinformatics/bty191'
 arch=('x86_64')
 url="https://github.com/lh3/${pkgname}"
 license=('MIT')
-depends=(zlib)
-provides=()
-source=("${pkgver}.tar.gz::$url/archive/refs/tags/v$pkgver.tar.gz")
-sha256sums=('f4c8c3459c7b87e9de6cbed7de019b48d9337c2e46b87ba81b9f72d889420b3c')
+depends=('zlib' 'glibc')
+provides=('minimap2-lite' 'sdust' 'libminimap2.so')
+source=("${pkgname}-${pkgver}.tar.gz::$url/archive/refs/tags/v$pkgver.tar.gz"
+        "hardening.patch")
+sha256sums=('f4c8c3459c7b87e9de6cbed7de019b48d9337c2e46b87ba81b9f72d889420b3c'
+            '792146de76e9003cccb429623eae715ab0349396eafda3001bdec940323e9356')
+
+prepare() {
+  # Add hardening flags for binaries
+  cd "${pkgname}-${pkgver}"
+  patch -p1 < "${srcdir}"/hardening.patch
+}
+
 build() {
   cd "${pkgname}-${pkgver}"
-  make
+  # Build all binaries
+  make extra
+
 }
 
 package() {
   cd "${pkgname}-${pkgver}"
-  install -Dm644 LICENSE.txt "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
-  install -Dm644 ${pkgname} "${pkgdir}/usr/bin/${pkgname}"
-  chmod +x "${pkgdir}/usr/bin/${pkgname}"
+  install -Dm644 LICENSE.txt "${pkgdir}"/usr/share/licenses/${pkgname}/LICENSE
+  install -Dm644 libminimap2.so "${pkgdir}"/usr/lib/libminimap2.so
+  install -Dm755 ${pkgname} "${pkgdir}"/usr/bin/${pkgname}
+  install -Dm755 minimap2-lite "${pkgdir}"/usr/bin/minimap2-lite
+  install -Dm755 sdust "${pkgdir}"/usr/bin/sdust
+  # install Manpages
+  install -Dm644 -t "$pkgdir/usr/share/man/man1" minimap2.1
+  # install test Data
+  install -d ${pkgdir}/usr/share/${pkgname}/test
+  cd test
+  for file in *; do
+    install -Dm644 "$file" "${pkgdir}"/usr/share/${pkgname}/test/"${file}"
+  done
```

Comment:
The previous copying using `cp -r` was shorter, but it's fine this way as well. There's missing double quotes on line 41 though.